runtime/doc/news.txt

```diff
@@ -393,6 +393,14 @@ release.
 • Configuring |diagnostic-signs| using |:sign-define| or |sign_define()|. Use
   the "signs" key of |vim.diagnostic.config()| instead.
 
+• |vim.diagnostic| functions:
+  - |vim.diagnostic.goto_next()| and |vim.diagnostic.goto_prev()| are
+    deprecated in favor of |vim.diagnostic.goto()|
```

Comment:
```suggestion
    deprecated in favor of |vim.diagnostic.goto()|. Use
    `vim.diagnostic.goto({count = 1})` and `vim.diagnostic.goto({count = -1})`, respectively.
```

it would be even better if we can guide users how they are supposed to migrate explicitly.

Reply:
The link is (more) important, though.

Reply:
Agreed that having a link to the new function is more valuable. The link provides the full documentation of the new API.